nilearn/input_data/nifti_masker.py

```diff
@@ -4,6 +4,7 @@
 # Author: Gael Varoquaux, Alexandre Abraham
 # License: simplified BSD
 
+import numpy as np
```

Comment:
Why is the numpy import necessary?

Reply:
May be I have forgotten to remove while removing sanity checks. I will clean up in next commit along with some smoke tests.

Thanks